pkg/server/storage/sqlite.go

```diff
@@ -50,6 +50,28 @@ func (s *SqliteStorage) QueryHostkeys() (ResultIterator, error) {
 	return &SqlResultIterator{Rows: rows}, nil
 }
 
+func (s *SqliteStorage) RecordGithubMapping(mapping map[string]string) error {
+	for ssoIdentity, githubUser := range mapping {
+		_, err := s.DB.Exec("INSERT OR REPLACE INTO github (ssoIdentity, githubUser) VALUES (?, ?)",
```

Comment:
just double-checking that this also works with SQLite

Reply:
yeah I tested it with both